lib/vagrant/compose/programmatic/cluster.rb

```diff
@@ -39,7 +39,6 @@ def initialize(name)
           @ansible_context_vars  = {}
           @ansible_group_vars    = {}
           @ansible_host_vars     = {}
-          @multimachine_filter   = ""
```

Comment:
ok, thanks, @multimachine_filter here is not necessary.
multimachine_filter functionalities is now implemented in config.rb